### ISSUE TITLE
Review the alt text for Credits page images and ensure that they adhere to WCAG

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2572]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Credits page.

#### Details
Developers identified that some alt texts in the Credits page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
UI/UX Design
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on to the Resources/Instructions section of this issue at the bottom where it says "Google sheet Tab for your page".
- [x] For each image on the Credits page identify if the alt text needs to be changed, or not, using the recourses below, and fill out the google sheet.
- [x] After completing auditing each image change your google sheet tabs [Status] to Audit Complete.
- [x] Review with Design Lead or another member of the team
- [x] Mark spreadsheet tab as QA (e.g., About Page - Audit QA)
- [x] Add note at bottom of issue saying its ready for QA and use the close with comment, so that it goes to the QA column

Product
- [x] Review the work and sign off on it if ready
   - [x] Mark spreadsheet tab as Complete (e.g., About Page - QA Complete)
   - [x] Add Ready for dev leads label
   - [x] Move to New Issue Approval Column

Development
- [ ] Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix
- [ ] Close this issue

#### Summary
If images provide more meaning = needs alt text
If images are decorative = null alt text (alt="")

### Resources/Instructions
[Credits page](https://www.hackforla.org/credits/)
[Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
[Decorative Images and alt text](https://www.w3.org/WAI/tutorials/images/decorative/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit?pli=1#gid=456130506)

**Comment by iilashi** [comment]
Progress: WIP
Blockers: None
Availability: A few hours a day
ETA: Saturday (1/22)

**Comment by IsaacDesigns** [comment]
@ilashi I believe you are correct in that the images should have alt text but I don't think it should be changed to what you are suggesting. The reason is because the current alt text gives more clarification of the images than the change you are suggesting. I'm also thinking about the edge case of what alt text would be most helpful for someone who is blind that uses a screen reader.

**Comment by iilashi** [comment]
@IsaacDesigns I definitely agree now that I look at it, I've went ahead and made the changes needed.

**Comment by Providence-o** [comment]
This issue is ready for QA

**Comment by ExperimentsInHonesty** [comment]
- Dev has taken the handoff and the new issue is https://github.com/hackforla/website/issues/3449

**Comment by JessicaLucindaCheng** [comment]
> * Dev has taken the handoff and the new issue is [Update Resource Cards On Projects Pages #3449](https://github.com/hackforla/website/issues/3449)

@ExperimentsInHonesty The issue you linked to isn't related to fixing the alt text on the credits page. Also, I don't think an epic and its issues were written for devs as the follow-up to this issue for alt text on the Credits page. So, I will put this back into the New Issue Approval column with a "ready for dev lead" label.

@kathrynsilvaconway Could you check if issues were written to fix the alt text on the credits page?

**Comment by bishrfaisal** [comment]
I QAed this sheet, it is ready for dev leads to make good first and second issues out of it.

**Comment by ExperimentsInHonesty** [comment]
I agree, the spreadsheet is ready for dev leads to make new issue from it.

**Comment by ExperimentsInHonesty** [comment]
This can be resolved with the creation of one issue to edit https://github.com/hackforla/website/blob/gh-pages/pages/credits.html line 32 to revise from
```
alt="{{ item[1].alt }}"
```
to
```
alt=""
```

**Comment by MattPereira** [comment]
- #3872
